src/network/message.rs

```diff
@@ -117,7 +118,18 @@ pub enum NetworkMessage {
     Pong(u64),
     // TODO: reject,
     // TODO: bloom filtering
-    // TODO: alert
+    /// BIP157 getcfilters
+    GetCFilters(message_filter::GetCFilters),
+    /// BIP157 cfilter
+    CFilter(message_filter::CFilter),
+    /// BIP157 getcfheaders
+    GetCFHeaders(message_filter::GetCFHeaders),
+    /// BIP157 cfheaders
+    CFHeaders(message_filter::CFHeaders),
+    /// BIP157 getcfcheckpt
+    GetCFCheckpt(message_filter::GetCFCheckpt),
+    /// BIP157 cfcheckpt
+    CFCheckpt(message_filter::CFCheckpt),
```

Comment:
What do you think about moving these to a separate enum, say... `network::message::ClientFilter`?